.rubocop.yml

```diff
@@ -962,6 +962,9 @@ Performance/CollectionLiteralInLoop:
   Enabled: false
 
 Performance/CompareWithBlock:
+  Enabled: false
```

Comment:
In addition to the 2 new cops being added, this existing cop `Performance/CompareWithBlock` seems to have been disabled by these changes. Would you please either reenable it or update the PR description to explain why it's a good idea to disable it?

Reply:
Oh shoot. Thank you, was testing something. Not supposed to be disabled.